vmm/src/vm.rs

```diff
@@ -517,6 +523,9 @@ struct DeviceManager {
 
     // PCI root
     pci: Arc<Mutex<PciConfigIo>>,
+
+    // List the presense of the signals with values from 1 to 64
+    signal_present: [bool; MAX_SIGNAL_VAL + 1],
```

Comment:
I think `interested_signals` or something would be a clearer name here

Reply:
Or perhaps `signal_handler_present`.

Reply:
As per suggestion from Sebastien, we don't need generic code for signaling. So, these are not relevant now and thus I removed it and associated implementation.